build.psm1

```diff
@@ -211,6 +211,10 @@ function Start-PSBuild {
         [switch]$NoPSModuleRestore,
```

Comment:
@rjmholt, your last commit had 32 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Experimental Feature: && and || operators - Feature-Enabled.Runtime error semantics.Returns a partial result with a terminating command
```powershell
Expected strings to be the same, but they were different.
Expected length: 30
Actual length:   24
Strings differ at index 0.
Expected: 'BAD,Test-TerminatingBadCommand'
But was:  'CommandNotFoundException'
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1: line 188
188:                 $_.FullyQualifiedErrorId | Should -BeExactly 'BAD,Test-TerminatingBadCommand'
```
Experimental Feature: && and || operators - Feature-Enabled.Runtime error semantics.Returns a partial result with a terminating command following ||
```powershell
Expected strings to be the same, but they were different.
Expected length: 30
Actual length:   24
Strings differ at index 0.
Expected: 'BAD,Test-TerminatingBadCommand'
But was:  'CommandNotFoundException'
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1: line 201
201:                 $_.FullyQualifiedErrorId | Should -BeExactly 'BAD,Test-TerminatingBadCommand'
```
Experimental Feature: && and || operators - Feature-Enabled.Runtime error semantics.Does not continue execution when terminating command comes first
```powershell
Expected strings to be the same, but they were different.
Expected length: 30
Actual length:   24
Strings differ at index 0.
Expected: 'BAD,Test-TerminatingBadCommand'
But was:  'CommandNotFoundException'
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1: line 214
214:                 $_.FullyQualifiedErrorId | Should -BeExactly 'BAD,Test-TerminatingBadCommand'
```
Experimental Feature: && and || operators - Feature-Enabled.Runtime error semantics.Does not continue execution when terminating command followed by ||
```powershell
Expected strings to be the same, but they were different.
Expected length: 30
Actual length:   24
Strings differ at index 0.
Expected: 'BAD,Test-TerminatingBadCommand'
But was:  'CommandNotFoundException'
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1: line 227
227:                 $_.FullyQualifiedErrorId | Should -BeExactly 'BAD,Test-TerminatingBadCommand'
```
Experimental Feature: && and || operators - Feature-Enabled.Runtime error semantics.Correctly turns a throw value into a job -- edge case
```powershell
Expected an exception, with FullyQualifiedErrorId 'System.Management.Automation.PSRemotingJob' to be thrown, but no exception was thrown.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\PipelineChainOperator.Tests.ps1: line 236
236:             } | Should -Throw -ErrorId 'System.Management.Automation.PSRemotingJob'
```